vpr/src/draw/intra_logic_block.cpp

```diff
@@ -400,7 +416,7 @@ static void draw_internal_pb(const ClusterBlockId clb_index, t_pb* pb, const ezg
             if (draw_state->draw_block_text) {
                 g->draw_text(
                     ezgl::point2d(abs_bbox.center_x(),
-                                  abs_bbox.top() - (abs_bbox.height()) / 15.0),
+                                  abs_bbox.top() - draw_coords->get_tile_height() * 0.01),
```

Comment:
Suggest tying this 0.01 to one of the const drawing values above, or making a new one.  It'll be hard to  find and update/align this 0.01 with other drawing offsets in future changes if you don't give it a name (or even better, compute it from the constants already present).